### DISCUSSION TITLE
How to notify renovate about my product updates?

### DISCUSSION THREAD

**Discussion by Im-Fran** [renovatebot/renovate#20408] [Request Help] [answered]
Hi! Is there a way for me to let renovate know about my product updates?

Like for example, when I push an update of https://github.com/TheProgramSrc/SimpleCoreAPI, I want to see PRs from renovate in my other projects that use this package, for example https://github.com/TheProgramSrc/SimpleCore-FilesModule that relies on SimpleCoreAPI to work.

Is there a webhook or rest api I could use? If my idea is not clear enough let me know :p

**Comment by rarkins** [comment, marked as answer]
I understand what you mean, but this is currently not possible

**Reply by Im-Fran** [reply]
Hmm, thanks. Is there any future updates that might contain this feature? I'm really looking into it.

**Reply by rarkins** [reply]
It's not planned, but it's of interest to me.

FWIW this is not really _renovate_ the open source CLI functionality, because a CLI has no REST API or ability to receive webhooks. This would be related to the Renovate solutions by my employer Mend, such as the github app.

First of all, we plan anyway to add a capability to listen/monitor to most popular registries for new package releases and then trigger higher priority jobs for all installed repos which we know to use those packages. If your package is public, then you wouldn't really need to send us a webhook becuase we'd hopefully know about it within 5-15 minutes of it getting published to e.g. Maven Central,

If it's a private package, it's a bit harder. We can't just let anyone from anywhere to send us webhooks about random packages - we'd need a way in which we issue credentials of sorts after you prove ownership of a package or scope.

**Reply by Im-Fran** [reply]
I understand. And do you guys plan on adding more maven repos to listen? Because currently I'm using sonatype oss and I've seen that since February 2021 they added the url s01.oss.sonatype.org, also my packages are posted as snapshots, and maven central only accept stable releases...

**Reply by rarkins** [reply]
Snapshots from non default registries are likely to be a lower priority
